Dockerfile

```
@@ -0,0 +1,14 @@
+FROM continuumio/miniconda3
+
+WORKDIR /workshop
+
+RUN /opt/conda/bin/conda install jupyter
+
+COPY environment.yml .
+RUN /opt/conda/bin/conda env create --file environment.yml
+
+COPY . .
+
+# Run within the container and allow access without token from localhost:8888
+CMD [ "conda", "run", "-n", "data_viz_workshop", "jupyter", "lab", \
+    "--allow-root", "--ip=0.0.0.0", "--no-browser", "--NotebookApp.token=''" ]
```

We can leave it with multiple per line, but can we just line this quote up with the one above?
```suggestion
CMD [ "conda", "run", "-n", "data_viz_workshop", "jupyter", "lab", \
      "--allow-root", "--ip=0.0.0.0", "--no-browser", "--NotebookApp.token=''" ]
```

---

@stefmolin Done 👍